course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -522,7 +527,7 @@
    "source": [
     "----\n",
     "\n",
-    "## 3.4 Exercise 3 <a id='exercise'></a>"
+    "## 3.4 Section Review Exercise <a id='exercise'></a>"
```

Comment:
Open to a better name for this ?

Reply:
I was thinking along the same lines.
I'm happy with this for now. If we come up with a better name we can change it later.